distribution/lib/Standard/Database/0.0.0-dev/src/Feature.enso

```diff
@@ -41,8 +41,8 @@ type Feature
        currently blocks getting a DB_Column from a DB_Table, but will soon refine to operations on DB_Column.
     Column_Operations
     ## PRIVATE
-       set operations on tables.
-    Set
+       set from expression operations on tables. Other set types should always be possible.
+    Set_From_Expression
```

Comment:
I don't understand why this is needed as a feature - conversion from an expression to a Column occurs internally with no need to go to dialect/connection?

Reply:
Removed